docs/products/opensearch/howto/import-opensearch-data-using-elasticsearch-dump.rst

```diff
@@ -0,0 +1,119 @@
+
+Dump OpenSearch index using ``elasticsearch-dump``
+==================================================
+
+It is a good practice to perform backups of your OpenSearch data to another storage service. This way you can access your data and restore it if something unexpected happens to it. 
+
+In this article, you can find out how to dump your OpenSearch data to an:
+
+* :ref:`Aiven OpenSearch <copy-data-from-os-to-os>`
+* :ref:`AWS S3 bucket <copy-data-from-os-to-s3>`
+
+To copy the index data, we will be using ``elasticsearch-dump`` `tool <elashttps://github.com/elasticsearch-dump/elasticsearch-dump>`__. You can read the `instructions on GitHub <https://github.com/elasticsearch-dump/elasticsearch-dump/blob/master/README.md>`_ in how to install it. From this library, we will use ``elasticdump`` command to copy the ``input`` index data to an specific ``output``. 
+
+.. note::
+
+    Make sure to have ``elasticsearch-dump`` `tool <elashttps://github.com/elasticsearch-dump/elasticsearch-dump>`__ installed for the next steps.
+
+.. _copy-data-from-os-to-os:
+
+Copying data from OpenSearch to Aiven OpenSearch
+------------------------------------------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* OpenSearch cluster as the ``input`` (can be in Aiven or elsewhere)
+* Aiven for OpenSearch cluster as the ``output``
+
+.. note::
+    
+    The ``input`` and ``ouput`` can be either an OpenSearch URL or a file path (local or remote file storage). In this particular case, we are using both URLs, one from an **OpenSearch cluster** and the other one from **Aiven for OpenSearch cluster**. 
+
+
+Here are some information you need to collect to copy your data:
+
+OpenSearch cluster:
+
+* OpenSearch cluster URL , for e.g. ``http://opensearch-url:9243/``
+
+Aiven for OpenSearch:
+
+* ``INDEX_NAME``: the name of the index that you aim to backup.
```

Comment:
isn't this the target index name, not the one you aim to backup? Also can i backup an index named `students` into an index named `stud` or do they need to share the same name? 
If it's the first case (names can be different) I would use two different names like `INPUT_INDEX_NAME` and `OUTPUT_INDEX_NAME`

Reply:
Good you mention! I had not tested this but I tested now and you can change the output index name (not necessarily to keep the same from the input one). I will make the changes.